zfsbootmenu/help-files/92/zfsbootmenu.7.ansi

```diff
@@ -136,7 +136,7 @@
 
     [1mzbm.hookroot=<hookspec>[0m
 
-      Tell ZFSBootMenu to attempt to read any early-setup, setup or teardown hooks from the
+      Tell ZFSBootMenu to attempt to read any early-setup, setup load_key or teardown hooks from the
```

Comment:
```suggestion
      Tell ZFSBootMenu to attempt to read any early-setup, setup, load_key or teardown hooks from the
```
You should just be running `releng/rst2help.sh` to render these help files based on `docs/man` sources.